corehq/apps/cleanup/management/commands/evaluate_couch_model_for_sql.py

```diff
@@ -0,0 +1,58 @@
+from collections import defaultdict
+
+import logging
+
+from django.core.management.base import BaseCommand
+
+from corehq.dbaccessors.couchapps.all_docs import get_all_docs_with_doc_types, get_doc_count_by_type
+from corehq.util.couchdb_management import couch_config
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+        Given a set of attributes and a couch document type, Iterates over all documents
+        and reports back on whether those attributes are ever null and what their largest
+        values are.
+    """
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            'doc_type',
+        )
+        parser.add_argument(
+            '-a',
+            '--attrs',
+            nargs='+',
+        )
+        parser.add_argument(
+            '-s',
+            '--db',
+            action='store',
```

Comment:
nit: `action` should be necessary here

Reply:
3288ad6623b5ef9b4af5037d240c85d91bfcdd3c